### ISSUE TITLE
Bot not working!

### ISSUE THREAD

**Issue by PDeception** [finndev/PokeBuddy#218]
Ninatic just updated their API, the bot wont work until its updated too (atleast, it doesnt work for me). Just saying, don't post tons of bug posts about it.

**Comment by Vanethir** [comment]
Won't work I think. :P Niantic Server seem to be down anyways right now. :)

**Comment by Lordpigg** [comment]
O ok

**Comment by lordofthereef** [comment]
Since I am ignorant to these things, is this an anti botting move they did or... what does i t mean?> :)

**Comment by PDeception** [comment]
> lordofthereef commented 17 seconds ago
Since I am ignorant to these things, is this an anti botting move they did or... what does i t mean?> :)

Server is just down. Probably not an anti-bot thing.

**Comment by igneousdaffodil** [comment]
Hopefully! I am dealing with i logged out 2 hours ago, took  a little break and moved down town from the area i was in last, and as i log in this is all thats happened thus far
[P - 16:08:37] PokeBuddy launching...
[P - 16:08:37] Version: 0.0.6.5
[P - 16:08:37] Info: Settings Window is being improved
[P - 16:08:37] 	 Added: (!) New settings window. Now, everything is configurable.
[P - 16:08:37] This bot is free to use. If you paid for it, you got scammed.
[P - 16:08:37] Feel free to donate to (PayPal): support@epicdevs.de (make sure to email me afterwards)
[P - 16:08:37] --------------------------------
[P - 16:08:54] Attempting to Login...
[P - 16:08:54] Logging in with method: Google as Auth Provider
[P - 16:09:28] Stopping...
[P - 16:09:44] Attempting to Login...
[P - 16:09:44] Logging in with method: PTC as Auth Provider
[P - 16:09:53] Stopping...
[P - 16:10:05] Attempting to Login...
[P - 16:10:05] Logging in with method: Google as Auth Provider
 

**team rocket strikes again**

**Comment by stban4ever** [comment]
and now?

**Comment by ShupengHu** [comment]
It seems that either server is down or something wrong in connection with server because I can log in but no pokestops existing at live map

**Comment by curtiola** [comment]
@ShupengHu yeah same for on my mobile, but ptc wont log in

**Comment by cynicalme** [comment]
http://ispokemongodownornot.com/

**Comment by Vanethir** [comment]
**New features for Pokemon GO (not PokeBuddy):**
Trainers, Pokémon GO has been updated to version 0.31.0:
- Avatars can now be re-customized from the Trainer profile screen
- Adjusted battle move damage values for some Pokémon
- Refined certain Gym animations
- Improved memory issues
- Removed footprints of nearby Pokémon
- Modified battle damage calculation
- Various bug fixes during wild Pokémon encounter
- Updated Pokémon details screen
- Updated achievement medal images
- Fixed issues with displaying certain map features
- Minor text fixes

**Comment by IamDeric** [comment]
Server is not down, i just catch some pokemon, but in real world. Did not check my bot yet.

**Comment by curtiola** [comment]
@Vanethir i've had 0.31.0 for a few days now, it may have something to do with them trying to roll it out in brazil today

**Comment by EaglesGPC** [comment]
Bot, just start, now...

**Comment by crilleaz** [comment]
@curtiola Has nothing to do with the Brazil release, my PikaBot is not working neither, it's because they changed the API.

**Comment by cynicalme** [comment]
Guys, there are two types of logins for the game. Google and PTC (Pokemon Trading Club). 

If you go to 

https://sso.pokemon.com/sso/login?locale=en&service=https://www.pokemon.com/us/pokemon-trainer-club/caslogin

and try to log in you'll get this message:

"Oops! It seems that Pokemon.com has had an unexpected error. Don't worry; the site administrators have already been notified and they'll try to have the problem fixed soon."

The bots, as far as I can tell, use both Google and PTC login procedures assuming you have a Google account to use. Since the PTC is experiencing issues, the bot will have issues. 

*Disclaimer - I don't know much about this stuff but I spent 15 minutes googling things.*

**Comment by SparkzZone** [comment]
thank you but I can log in on blue stacks and play but the bot wont load pokestops but does log in

**Comment by sdkrocks** [comment]
Same thing for me I can login and all but no walking or pokestops load. I can catch pokemon on my regular account, although.

**Comment by Sylencer90** [comment]
Works fine on my bluestacks, just upgraded some of my 100 IV's. It's probably some API update/change which causes the bot to mess up.

**Comment by SparkzZone** [comment]
ye ill just check back tomorrow

**Comment by Lesepp** [comment]
So it means we will not have the bot working for at least next 10h until developer helps us out?
Just found out about this magical tool and now can´t wait to try it out. 
Anyone knows when it sould start working again?
Thanks!

**Comment by PDeception** [comment]
When the developer has time to update the API.

**Comment by SkyVoidBringer** [comment]
On that note, who knows when the devs hop on and have a look at what's going on with their bot?

**Comment by lordofthereef** [comment]
The main dev seams to be active constantly so...

**Comment by LuzgGO** [comment]
Pokemon go IS working, but the bot don't

**Comment by PDeception** [comment]
Dev is on almost every night to check out the feedback.

**Comment by PDeception** [comment]
> Pokemon go IS working, but the bot don't

Pokemon Go has updated their API. The bot will not work until the API on the bot is also updated.

**Comment by Thrallix** [comment]
It has nothing to do with the API, as said.
The bot is currently disfunctional due to PokemonGo's server issues.

**Comment by SkyVoidBringer** [comment]
If that's the case then why can everyone log in and play like normal on their phones and other emulators?

**Comment by Thrallix** [comment]
Incase you had not realized, everything works except for Pokémon, and Pokéstops.

**Comment by cynicalme** [comment]
"Oops! It seems that Pokemon.com has had an unexpected error. Don't worry; the site administrators have already been notified and they'll try to have the problem fixed soon."

https://sso.pokemon.com/sso/login?locale=en&service=https://club.pokemon.com/us/pokemon-trainer-club/caslogin

The message you get when trying to log into via PTC. The PTC is the primary "gateway" used by bots. Since it's having issues, the bots will have issues.

**Comment by sWAGrOGER** [comment]
It seems like the bot isn't able to find the Pokestops as everything else works. 
This will be because Pokemon TC keeps going up and down. If you google the servers you can watch it go on and off every few minutes. They could have changed the API but I highly doubt it because they would have to roll out a new version of the game so we could all have the updated API in game normally. 

It would be so much easier if the DEV released the source code. Id make this app so much better yet I don't want to replicate it as the DEV made it fair and square.

**Comment by Thrallix** [comment]
So start your own?

**Comment by sWAGrOGER** [comment]
Nope no point. It would be better to help others improve.

**Comment by Thrallix** [comment]
You probably wouldn't even know how.

**Comment by PDeception** [comment]
Yeah. I dont know man. If you could improve his so much, just make your own "improved" version, instead of leeching off of others code, haha.

**Comment by sWAGrOGER** [comment]
What because I won't make my own? I'm leeching lol no it's called helping others out. It's not that it could be improved its that other functions could be added while he makes the framework. Get a grip and stop trying to pick and argument.

**Comment by Thrallix** [comment]
The fact that you want his sourcecode, with probably no skill at all, is the fact that's so strange.
If you do have skills, show us your portfolio?

If not, stop asking for his sourcecodes.

**Comment by dallashowell89** [comment]
sWAGrOGER = douche.  Make your own or shut up.  You couldn't code a module to save your life.  I can tell by your stupid remarks.  Code TD.

**Comment by sWAGrOGER** [comment]
I didnt ask for his source code I said it would be better if he released it. Learn to read and nope don't know how to code do I? forgot soz bro I'll just go give up on life. Fucking losers.

**Comment by Thrallix** [comment]
Show me some of your work then?
I bet you don't have one shitty piece you can show us.

**Comment by crilleaz** [comment]
I think its time for a rapbattle.

**Comment by sWAGrOGER** [comment]
and why would I need to prove myself to you? you little keyboard warrior you.

**Comment by SkyVoidBringer** [comment]
Rage 

Petty arguments are a waste of time

**Comment by dallashowell89** [comment]
If your so fucking smart.  Decompile it and reverse engineer it and then put your "improvements to it"

**Comment by Thrallix** [comment]
You can't decompile .exe's lmao...

**Comment by sWAGrOGER** [comment]
Haha idiot need some heal for that burn?

**Comment by Thrallix** [comment]
You're the idiot here...

**Comment by chrizfilmz** [comment]
the thread is not about how dumb swagor is its about the bot not working so stop it

**Comment by SkyVoidBringer** [comment]
Omg guys just shut up you all kids or something?

**Comment by crilleaz** [comment]
@Thrallix sure you can decompile .exe's.

**Comment by Thrallix** [comment]
Not with good results and usuable codes.

**Comment by Vanethir** [comment]
Will cleanup and let us get to topic, please. :)

**Comment by cynicalme** [comment]
http://pokemongo.nianticlabs.com/en/post/aug2016-update/

There's your update boys.

**Comment by Thrallix** [comment]
That's not from right now, I've seen that a few hours ago aswell.

**Comment by ykz89** [comment]
I believe that's the same message they posted on their Facebook page earlier.

**Comment by Goodguy18** [comment]
thats the info of the update that i already got tho removed steps etc , its the update that came out like couple days ago

**Comment by makleikle** [comment]
guys is the game without the bot working???? maybe its the game not the bot

**Comment by Goodguy18** [comment]
the game is working ... its just the bot somehow..

**Comment by makleikle** [comment]
fk :(

**Comment by gibby71** [comment]
I tested with a different bot and it is facing the same issue.  Looks like some issue or change at Ninatic.

**Comment by makleikle** [comment]
i think the problem is with the map

**Comment by makleikle** [comment]
i cant log in my google account on pokemon go maybe the bot scamed us???

**Comment by Goodguy18** [comment]
yepp thats it theres something fishy about this , i think just bot needs a lil update

**Comment by makleikle** [comment]
come on i just found out for this bot and it got already patched shit

**Comment by makleikle** [comment]
never lucky

**Comment by PDeception** [comment]
Lol. Guys. The servers are having problems, and the API may of been updated. The bot is not patched, it just isnt up to date. Wait a couple hours.

**Comment by Goodguy18** [comment]
and btw i dont think this bot will scam alot ppl use it

**Comment by curtiola** [comment]
don't worry guys, Vanethir and Finndev are very active, this will most likely be fixed today

**Comment by makleikle** [comment]
ok

**Comment by Goodguy18** [comment]
ohh im not worried just trying to help out kinda i got time man dont worry :)

**Comment by Goodguy18** [comment]
flippin got me lvl 20 one day no ban nothing :D

**Comment by curtiola** [comment]
im on the end of day 3, level 31 xD

**Comment by Goodguy18** [comment]
damnnn sonnn!! did you caught any sick pokemon? :o

**Comment by crilleaz** [comment]
Just evolved my 4th Gyarados.

**Comment by makleikle** [comment]
i caught a 2,5k cp

**Comment by PDeception** [comment]
lol, I bot accounts up to lv 30, and then I sell them for 40-50 dollars to people who dont know about the botting.

**Comment by cynicalme** [comment]
@crilleaz what's the CP jump from magikarp to Gyarados?

**Comment by curtiola** [comment]
a dragonite at 47CP hahaha, nothing super special really, a 2090 snorlax and a bunch of exeggutor/vaporeon/flareon around 2100

**Comment by makleikle** [comment]
same right here xD haha

**Comment by Goodguy18** [comment]
sick it does depend ur lvl what pokemon you see right?

**Comment by curtiola** [comment]
avg cp jump from magikarp is about 10.5, my 188 turned into 1945 gyarados, i now have a 206 magikarp i plan to power up soon xD

**Comment by makleikle** [comment]
in cp yes not on rarity

**Comment by Fenderfix** [comment]
But I can't found Pokéstop or Pokémon in this moment with the Bot... Is this problem only mine?

**Comment by Goodguy18** [comment]
so , we gotta keep switching spots , what are any good spots i live in holland all i get is fricking pidgeys and rattata's sometimes a growlithe or so but nothing special :(

**Comment by curtiola** [comment]
@Goodguy18 yeah being higher level makes a big difference, from finding eevee at 600-700 at level 25, now finding eevee around 800-900 at 31

**Comment by makleikle** [comment]
Sydney is good

**Comment by curtiola** [comment]
@Goodguy18 i always use these coords 40.77844539228035,-73.96852254867554 central park NY

**Comment by PDeception** [comment]
The spots I use are in japan, and in Britain around the Big Ben, and then Central Park in NY.

**Comment by makleikle** [comment]
nah not good
go Sydney full lure

**Comment by Goodguy18** [comment]
ohh sht right central park , forgot that one thats the one i wanted to use man shoutout to you man!! :D

**Comment by makleikle** [comment]
also alot of dantinies

**Comment by alexcaputo** [comment]
Just tried it here is Australia and no issues with server, bot still not finding pokestops though

**Comment by Goodguy18** [comment]
why the hell is central park not good you ever seen the lures there? :o :(

**Comment by Fenderfix** [comment]
But I can't found Pokéstop or Pokémon in this moment with the Bot... Is this problem only mine?

**Comment by sdkrocks** [comment]
Or San Francisco, lot of dratinis their and magikarp

**Comment by curtiola** [comment]
i really like central park, but dratini are good, ill give sydney a shot sometime, thanks @makleikle

**Comment by Goodguy18** [comment]
no its not bot needs a lil update or the servers are unstable.

**Comment by sdkrocks** [comment]
@Fenderfix Every bot is having that problem currently, read up above in the thread

**Comment by makleikle** [comment]
yeah i agrre
San francisco and Sydnay best spots yet

**Comment by sdkrocks** [comment]
^^^

**Comment by Goodguy18** [comment]
i tried ehmm santa monica pier but it kept saying no pokestops around

**Comment by makleikle** [comment]
yes same here

**Comment by sdkrocks** [comment]
Bot is not working currently, probs why.

**Comment by makleikle** [comment]
game works fine bots map not at all

**Comment by Goodguy18** [comment]
ill go with san francisco , any1 got some awesome coords for me like right in the middle , appreciate it alot man <3

**Comment by curtiola** [comment]
yeah right now we need to wait for Finndev to update the bot, it seems niantic changed the API, probably won't be too long before Pokebuddy works again @Goodguy18

**Comment by alexcaputo** [comment]
Can you guys please leave this for the issue and stop posting about pokemon stats and game stuff, this is an issue

**Comment by sdkrocks** [comment]
The devs just need to update the bot to work again

**Comment by Goodguy18** [comment]
i know @curtiola i got time just came from work and it did a hell of a job man

**Comment by lmAshh** [comment]
top 7 for me= Arcanine 2160, Dragonite 2123, Arcanine 2045. Machamp 1890, Charizard 1883. Arcanine 1865, Snorlax 1702..

thanks to this bot i got a lot of stardust and ranked me up to level 27

**Comment by Goodguy18** [comment]
any awesome coords at alot pokestops and pokemon for san francisco? id appreciate it :D

**Comment by Suzano-Sho** [comment]
This post needs to be stickied lol...I predict many more threads like this cropping up in the coming hour...

**Comment by Goodguy18** [comment]
kinda in the middle somewhere there or give me some coords where u guys start at in san francisco :)

**Comment by makleikle** [comment]
try Sydney its a bit better for the bot bot for fake gps san francisco is better

**Comment by lmAshh** [comment]
@Goodguy18 37.7999192,-122.414526

**Comment by Kromes** [comment]
Santa Monica is not bad too !!

**Comment by Suzano-Sho** [comment]
@Goodguy18 don't have coords yet but the smaller park in the northern part of San Fran (along the river) was a dope Squirtle farming spot...I hear it's been changed to a Charmander farm now...

**Comment by Goodguy18** [comment]
thanks ash u got lots good pokemons there? :) :)

**Comment by cynicalme** [comment]
I have been using Disneyland in Anaheim. At lvl 25 I've been getting 800-900cp Growlithes.

**Comment by Synvir** [comment]
Everyone its every bot out there. Gobot , Necro etc. The API has been changed only on the pokemon references and stops. Everything works in game and it wasn't pushed as a official patch. Niantic has made inside comments about trying to prevent this. The API will need to be decoded again then simply updated to the bots. it'll take a day.  Not server issues... not sure how that was even decided

**Comment by lmAshh** [comment]
i havent used san fran recently just LA get a ton of charmanders and 900+ growlithes :)

**Comment by Goodguy18** [comment]
im kinda hyped to know whats all in san francisco :D

**Comment by makleikle** [comment]
i found a mew in sf

**Comment by makleikle** [comment]
3984cp

**Comment by ghettoprincess1** [comment]
Any idea when all the bots will go up?

**Comment by makleikle** [comment]
maybe 2morw

**Comment by Goodguy18** [comment]
some1 said it could take up to a day so

**Comment by Synvir** [comment]
about a day or 2 id imagine. have to change the api references

**Comment by Goodguy18** [comment]
awww man well aight np i guess :\

**Comment by PokemonEX** [comment]
Hey guys, do you like my user name?

**Comment by PokemonEX** [comment]
how fast does the developer update pokebuddy?

**Comment by Rab89** [comment]
Not sure just got the Bot today

**Comment by curtiola** [comment]
quickly by my standards, i couldn't tell you how long it will be before Finndev fixes this, but i know he's no slacker

**Comment by Goodguy18** [comment]
lets say a day or maybe to , so what its free

**Comment by PokemonEX** [comment]
is Vanethir and Finndev the same person? I donated some money to their paypal address found on the PokeBuddy. The money went to them right?

**Comment by Goodguy18** [comment]
two*

**Comment by curtiola** [comment]
@PokemonEX i doubt they're the same person lol and i assume it got there, did you send an email to address they provided in the console?

**Comment by Goodguy18** [comment]
ohh question , does the bot auto update or do we need to redownload? :)

**Comment by Kromes** [comment]
sometime just reload

**Comment by ProzanRaul** [comment]
Anyone who is softbanned,have already tried to spin a pokestops 41 times on the mobile app,to see if that still works on unbaning yourself?

**Comment by PokemonEX** [comment]
so i got "detected Cheater" from PokeAdvisor. What happens now?

**Comment by Rab89** [comment]
Did that ever work??

**Comment by Vanethir** [comment]
@PokemonEX He is not me or the other way around. :smile: 
He is working on the code while I'm here trying to solve your problems and giving updates about stuff I hear. 
Make sure to write him an email. I'm sure he will highly appreciate it. :heart: 

@Goodguy18 Usually restart the app to fetch new downloads is enough. If it doesn't work try to reinstall. Not working yet anyways. :)

**Comment by PokemonEX** [comment]
i appreciate the great customer service!

**Comment by alexcaputo** [comment]
Please don't post in this issue about separate issues... let the guys do their job and only post about issue relating to the API issue in the description

**Comment by ProzanRaul** [comment]
If anyone tried to spin the pokestop for 41 times please tell me

**Comment by Rab89** [comment]
time to hatch some eggs me thinks

**Comment by UknownAX** [comment]
Can someone leave the coordinates of the best spot at Sydney and any other place that could be the best.

**Comment by Suzano-Sho** [comment]
I can verify that spinning a pokestop 40 times un-softbans you...did so this morning...

**Comment by Goodguy18** [comment]
you also got quickspin it does it for you and yup it works

**Comment by Suzano-Sho** [comment]
Just to clarify, you have to spin the pokestop and then tap the X every time...

**Comment by Goodguy18** [comment]
just use quickspin 1.1 or so

**Comment by curtiola** [comment]
@PokemonEX I have no idea, where does it say detected cheater on pokeadvisor?  I'm trying to check my account lol

**Comment by IAmYawz** [comment]
I have 6 Lapras but no Snorlax :(

**Comment by Goodguy18** [comment]
i got 3 snorlax's 2 lapras's

**Comment by capfive** [comment]
Guys there are people watching this thread for updates, PLEASE stop posting stuff unrelated to the issue

**Comment by Goodguy18** [comment]
got it boss

**Comment by WuhDuhFuxUp** [comment]
http://pkmngotrading.com/wiki/Nests A good site for the best coordinates for what you're looking for.

**Comment by finndev** [comment]
I am aware of the issue and am working on fixing the issue. ETA is 1-6hrs
Sorry for the inconveniences.

**Comment by Goodguy18** [comment]
damnn son , sorry? dude your hell quick man take your time i will watch this thread when its done :D

**Comment by Vanethir** [comment]
Will cleanup a lil bit. Use #227 if you want to keep discussing. Will open it for more topic related comments/updates once I finished cleaning up. :)

**Comment by SimbaModz** [comment]
So when do u guys think the api will be updated like today, tomorrow, few hours?

**Comment by iluminami** [comment]
cant live without the bot please xD thank you for wasting your time in our lives xD really. thanks! you do a great job

**Comment by Vanethir** [comment]
@SimbaModz as mentioned above: ETA between 1:22 - 7:22 CEST

**Comment by SimbaModz** [comment]
@Vanethir thank you do we'll be updated when its fixed on this post of if theres a different one can u link me to it i'd appreciate it :)

**Comment by TheTarzanGTR** [comment]
Va a haber que descargar un nuevo PokeBuddy o los cambios se van a aplicar a la versión actual?

**Comment by ElDingo** [comment]
Thanks For working On This So Quickly, Me and My Friend Are Really Enjoying This Application!

**Comment by Goodguy18** [comment]
devv is fast for sure im sure hes gonna fix this with 4 fingers in his nose lol xD

**Comment by ElDingo** [comment]
LOL

**Comment by SimbaModz** [comment]
@ElDingo its fixed?

**Comment by curtiola** [comment]
not yet

**Comment by dannykyhung** [comment]
Thank for the developer hard work, hope it will work as soon as possible

**Comment by makleikle** [comment]
@Vanethir im thinking of donated because of you fast response and your activity but the problem is that i leave in greece and i cant donate fking idiot laws i cant send money outside country this suck kinda but you worth some money hard work must be rewarded sad i cant doante im a programmer too so i understand you  :+1:

**Comment by makleikle** [comment]
meant to say donating xD lol

**Comment by SirButtFux** [comment]
I sent a donation and I was unsure whether you guys got my email or not.

**Comment by oOBlueArrowOo** [comment]
ughhhhhhhhh I need to cheat lol

**Comment by Vanethir** [comment]
Since there are a lot of people asking: It's not directly related to PokeBuddy, but I'm pretty sure that the results of this project will be used for PokeBuddy (as well as for almost any other project/bot).

[PokemonGoDEV](https://www.reddit.com/r/pokemongodev/comments/4w1cvr/pokemongo_current_api_status/)

As long as there are no results from these amazing devs working hard on this problem I wouldn't expect any ETAs or updates here as well. :)
